package.json

```diff
@@ -97,7 +97,8 @@
     "lodash": "^4.17.21",
     "node-cache": "^5.1.2",
     "ramda": "^0.28.0",
-    "typechain": "^8.1.0"
+    "typechain": "^8.1.0",
+    "@openzeppelin/contracts": "4.7.3"
```

Comment:
Are you using these anywhere?

Reply:
yes, dpm is using this for cloning

Reply:
Ah, you put the contracts in our repo as well. Alright